releases.yaml

```diff
@@ -46,27 +48,27 @@ core_lts:
 
 checksums:
   desktop:
+    '24.10': '489079483487f92ad0d2f3d4b6c88a7b197969eb286b277534047920854a8b03 *ubuntu-24.10-desktop-amd64.iso'
     '24.04.1': 'c2e6f4dc37ac944e2ed507f87c6188dd4d3179bf4a3f9e110d3c88d1f3294bdc *ubuntu-24.04.1-desktop-amd64.iso'
-    '23.10.1': '3b6c5275366d02160554fa5703add462da3b8ce9be1749f8806e8dbbffaa2b5a *ubuntu-23.10.1-desktop-amd64.iso'
```

Comment:
Not sure why are we removing checksums for previous release?

Reply:
I copied that release.yaml from ubuntu.com so I assume we don't allow download directly from the websites themselves. We only provide the latest and LTS